memory-bank/native-runtime/04-PROPOSED-ARCHITECTURE.md

```
@@ -0,0 +1,713 @@
+// @deprecated — re-exports our native implementations under AI SDK names
+export const generateText = nlGenerateText;
+export const streamText = nlStreamText;
+```
+
+---
+
+## 4.8 What we delete (when done)
+
+After all phases complete, these get removed:
+
+### From `package.json` dependencies
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)